app/views/idv/shared/_document_capture.html.erb

```diff
@@ -13,6 +13,7 @@
   status_endpoint: FeatureManagement.document_capture_async_uploads_enabled? ?
     idv_doc_auth_step_path(step: :verify_document_status) :
     nil,
+  status_poll_interval: Figaro.env.poll_rate_for_verify_in_seconds.to_i * 1000,
```

Comment:
WDYT of making this `status_poll_interval_ms`?

Reply:
> WDYT of making this `status_poll_interval_ms`?

That seems fine to me. Did you have the same opinion about how the value is named and passed around once it's initialized within the React application? I don't feel strongly one way or the other, but I usually consider milliseconds to be the de facto standard unit of time durations in JavaScript.

Reply:
Leaving the JS property name as-is seems fine to me, I just figured that since we're doing a units conversion here it would be clearer if we specified what we were converting it to

Reply:
Renamed in 8cafb4a.